x-pack/plugins/security_solution/public/cloud_security_posture/components/entity_insight.tsx

```diff
@@ -60,6 +65,28 @@ export const EntityInsight = <T,>({
 
   const isVulnerabilitiesFindingForHost = hasVulnerabilitiesFindings && fieldName === 'host.name';
 
+  const { signalIndexName } = useSignalIndex();
+
+  const { data: alertsData } = useQueryAlerts({
+    query: buildEntityAlertsQuery(fieldName, name, 500),
```

Comment:
we load only first 500. But as I wrote in another comment, we don't need the actual alerts, only the aggregation by severity